docs/hr/content/docs/data_integrations/duckdb.md

```diff
@@ -1,9 +1,149 @@
 # DuckDB
 
-Connecting:
+`superduperdb` supports DuckDB via the [`ibis` project](https://ibis-project.org/).
+With `superduperdb`, queries may be built which conform to the `ibis` API, with additional 
+support for complex data-types and vector-searches.
+
+## Setup
+
+The first step in working with DuckDB, is to define a table and schema.
+
+```python
+from superduperdb.backends.ibis.query import Table
+from superduperdb.backends.ibis.field_types import dtype
+from superduperdb import Schema
+
+db = superduper('duckdb://test.dbb')
+
+t = Table(
+    'my_table',
+    primary_id='id',
+    schema=Schema(
+        'table-schema',
+        fields={'id': dtype(str),
+                'custom_text': dtype(str),
+                'custom_integer': dtype(int)},
+    )
+)
+
+db.add(t)
+```
+
+## Inserting data
+
+Table data must correspond to the `Schema` for that table:
```

Comment:
When inserting data to DuckDB, the pandas  dataframe or source table must correspond to the defined schema  for that database table